### DISCUSSION TITLE
CustomActions .NET 6 support in WIX v4

### DISCUSSION THREAD

**Discussion by adhip94** [wixtoolset/issues#7478] [Questions] [answered]
Why does the Heatwave CustomActions project template not give an option to select a framework of .NET 6 or above?

![image](https://user-images.githubusercontent.com/20703483/237044650-e9176275-f681-4b74-a30a-299d91abc286.png)

**Comment by barnson** [comment, marked as answer]
Because DTF doesn't support .NET Core.

**Reply by nickhoeferpickpro** [reply]
then how do we run custom actions for .net core?

**Reply by robmen** [reply]
@nickhoeferpickpro please stop spamming every question about this topic. Be respectful of the people who volunteer their time to answer questions.

If you want to make demands for answers then become a customer and [purchase a support contract](https://www.firegiant.com/services/support-programs/).

**Reply by nickhoeferpickpro** [reply]
spamming?
I'm new to wix so I don't yet understand some of the nuances. 
I mean no disrespect.

**Reply by robmen** [reply]
Please do not go to every question on the topic in our Discussions and ask the exact same follow-up question. That will trigger notifications on every thread you respond to for people who follow the Discussions. You're creating noise and reducing the chances that volunteers will choose to interact with you.
